google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -420,6 +432,9 @@ public void setAutomaticRestart(Boolean automaticRestart) {
 
   @Override
   public String getDevAppserverLogLevel() {
+    if (StringUtil.isEmpty(settings.getDevAppserverLogLevel())) {
```

Comment:
Are these empty checks for the case where the values are unset, or when they are explicitly set to empty by the user?

If the first, we should set the defaults on AppEngineModelSettings instead. Although if the values are unset, their value is probably `null`.